main.cpp

```diff
@@ -32,16 +36,46 @@ int main()
 
         // smartHome.AddToXml("../data.xml");
 
-        smart_home::LightSensor light("Light", 600);
-        smart_home::AcUnit ac("AC");
+        smart_home::UniquePtr<int> ptr1(new int(1));
+        smart_home::UniquePtr<int> ptr2(new int(3));
+        
+        smart_home::UniquePtr<int> arr(new int[3]{1,2,3});
```

Comment:
can you add a new value here:
`smart_home::UniquePtr<int> ptr3 = std::nullptr`
and tell me if it works?

Reply:
I did it and it works